libcodechecker/report.py

```diff
@@ -136,7 +134,6 @@ def compare_ctrl_sections(curr, prev):
 
             for i, section in enumerate(event_sections):
                 loc = section['location']
-                line_num = loc['line']
```

Comment:
This change here is @gyorb's jurisdiction. Is this a good catch on a left-behind unused variable, or did we miss something last time this file was touched?